examples/walk-mode/index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en" xml:lang="en">
+
+<head>
+  <title>Walk Mode</title>
+
+  <!-- Live reload script -->
+  <script src="http://localhost:35729/livereload.js?snipver=1"></script>
+
+  <script type="module" src="main.js"></script>
+
+  <link rel="stylesheet" href="https://unpkg.com/@vertexvis/viewer@0.19.x/dist/viewer/viewer.css" />
+  <script type="module" src="https://unpkg.com/@vertexvis/viewer@0.19.x/dist/viewer/viewer.esm.js"></script>
+  <script nomodule src="https://unpkg.com/@vertexvis/viewer@0.19.x/dist/viewer.js"></script>
```

Comment:
We're planning on releasing the package shortly, right?

Reply:
Yep! I'll probably plan on releasing after the work in https://vertexvis.atlassian.net/browse/PLAT-2433